lib/ddq-backend-mysql.js

```diff
@@ -3,20 +3,23 @@
 const EMIT_DATA = "data", EMIT_ERR = "error";
 
 module.exports = (configValidation, crypto, EventEmitter, mysql, timers) => {
-    var cycleCounter, setRequeued;
+    var setRequeued;
 
     /**
      * Updates heartbeatDate of a specific record and calls the callback that is
      * passed in.
      *
+     * This makes it so that the restore function does not think we have been
+     * disconnected and makes the message available to be consumed again.
+     *
      * @param {Object} ddqBackendInstance
      * @param {string} recordId
      * @param {Function} callback
      */
     function heartbeat(ddqBackendInstance, recordId, callback) {
         ddqBackendInstance.connection.query(`
             UPDATE ??
-            SET heartbeatDate = NOW()
+            SET heartbeatDate = NOW(3)
```

Comment:
I see, so this makes it more precise for the deadlocking. Could you add a comment about this? After a quick google I couldn't find it here http://www.w3schools.com/sqL/func_now.asp and didn't figure it out until I tested it.

Makes sense though